YoutubeExplode/Videos/Streams/StreamClient.cs

```diff
@@ -31,10 +32,40 @@ public StreamClient(HttpClient http)
         _http = http;
         _controller = new StreamController(http);
     }
+    
+    private async Task<PlayerSourceExtractor?> TryGetPlayerSourceAsync()
+    {
+        var iframeContent = await _http.GetStringAsync("https://www.youtube.com/iframe_api");
+
+        var version = Regex.Match(iframeContent, @"player\\?/([0-9a-fA-F]{8})\\?/").Groups[1].Value;
+        if (string.IsNullOrWhiteSpace(version))
+        	return null;
+
+        var source = await _http.GetStringAsync($"https://www.youtube.com/s/player/{version}/player_ias.vflset/en_US/base.js");
+
+        return PlayerSourceExtractor.Create(source);
+    }
```

Comment:
One last thing, can you move this to `StreamsController` please? I think that's a better place for it